src/ceph-volume/ceph_volume/util/prepare.py

```diff
@@ -334,6 +334,30 @@ def _link_device(device, device_type, osd_id):
 
     process.run(command)
 
+def _validate_device(device, excepted_device_type, osd_uuid):
```

Comment:
```suggestion
def _validate_bluestore_device(device, excepted_device_type, osd_uuid):
```

A bit of a nit, but names being important and all...

Reply:
Fair enough!